src/executables/master_heartbeat.c

```diff
@@ -2737,9 +2761,11 @@ hb_cluster_load_group_and_node_list (char *ha_node_list, char *ha_replica_list)
 	}
     }
 
-  if (hb_Cluster->myself == NULL)
+  if (hb_Cluster->state == HB_NSTATE_REPLICA && hb_Cluster->myself == NULL)
     {
-      MASTER_ER_LOG_DEBUG (ARG_FILE_LINE, "cannot find myself. \n");
+      sprintf (err_string, "In replica mode, (%s) must be in the %s", hb_Cluster->host_name,
```

Comment:
Nit.
```suggestion
      sprintf (err_string, "The replica node(%s) must be specified in the %s", hb_Cluster->host_name,
```

Reply:
thank you.
Adding "specified" word is good.
But, I think it would be clearer to use the words "in replication mode" to let the user know which mode was set in the error message.

Reply:
OK, I agree!